### PR TITLE
Image and input components added

### PR DESCRIPTION
**Pull request by sakshi006**

### CODE REVIEW

File: components/Image/Image.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+            Images play an important role in any webpage. CSS plays a good role to control image display. You can set the following image properties using CSS. The border property is used to set the width of an image border. The height property is used to set the height of an image.
+            <br />
+            <br />
+          </p>
+
+          <div class="section section--type-one">
+            <h4>Images</h4>
+            <img src="../../assets/avatar/card-image.jpg" alt="img" class="image basic"/>
+
+            <img src="../../assets/avatar/book.jpg" alt="img" class="image basic"/>
```
Comment (kirti2602) [inline components/Image/Image.html:74]: Everything looks fine, just you can try reducing the size of this image for faster loading

**Review by kirti2602 — APPROVED** [review]
Looks good to me.

File: components/Image/Image.html (hunk 1 above)
Comment (sakshi006) [reply components/Image/Image.html:74]: > Everything looks fine, just you can try reducing the size of this image for faster loading

Got it.

**Review by Asuraking1n — COMMENTED** [review]
Looks cool to me

**Pull request merged**